### PR TITLE
Support any quotes in release.py

### PR DESCRIPTION
**Pull request by goosemania**
[noissue]

### CODE REVIEW

File: templates/github/.ci/scripts/release.py.j2 [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import argparse
 import json
+impore re
```
Comment (mdellweg) [inline templates/github/.ci/scripts/release.py.j2:5]:
```suggestion
import re
```

Comment (goosemania) [reply templates/github/.ci/scripts/release.py.j2:5]: ah jo, they would say in cz

**Review by mdellweg — APPROVED** [review]

**Comment by fao89** [comment]
@goosemania is something blocking the merge?

**Comment by goosemania** [comment]
> @goosemania is something blocking the merge?

Nope :)

**Pull request merged**